db/rdb/rdb.go

```diff
@@ -187,13 +194,13 @@ func (d *Driver) CloseDB() (err error) {
 }
 
 // GetByPackName select OVAL definition related to OS Family, osVer, packName
-func (d *Driver) GetByPackName(osVer, packName, arch string) ([]models.Definition, error) {
-	return d.ovaldb.GetByPackName(d.conn, osVer, packName, arch)
+func (d *Driver) GetByPackName(family, osVer, packName, arch string) ([]models.Definition, error) {
+	return ovalMap[family].GetByPackName(d.conn, osVer, packName, arch)
```

Comment:
This may cause nil panic. The existence of key should be checked.

Reply:
Thank you! Fixed